src/cli/cli-config.json

```diff
@@ -3,33 +3,33 @@
     "description": "Knative CLI tool",
     "vendor": "Red Hat, Inc.",
     "name": "kn",
-    "version": "1.7.1",
-    "versionRange": "1.7.1",
-    "versionRangeLabel": "v1.7.1",
+    "version": "1.8.1",
+    "versionRange": "1.8.1",
+    "versionRangeLabel": "v1.8.1",
     "versionLocalBuildRange": 20210101,
```

Comment:
@lstocchi Do we know why we need this field?

Reply:
it was born to deal with local build but it seems it was never updated https://github.com/redhat-developer/vscode-knative/blob/main/src/cli/cli-config.ts#L93

Reply:
I think we can remove it and even comment out the code logic. Just one less thing to check and maintain with the version.